src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -239,6 +240,8 @@ public final void setAllowSamelineMultipleAnnotations(boolean allow) {
         return new int[] {
             TokenTypes.CLASS_DEF,
             TokenTypes.INTERFACE_DEF,
+            TokenTypes.PACKAGE_DEF,
+            TokenTypes.ENUM_CONSTANT_DEF,
```

Comment:
@pbludov @romani Any reasons why these shouldn't be default?

Reply:
`ENUM_CONSTANT_DEF` definitely should be.
For `PACKAGE_DEF` I think it is better to add.